hadoop-ozone/ozone-manager/src/main/java/org/apache/hadoop/ozone/om/ratis/OzoneManagerStateMachine.java

```diff
@@ -136,6 +137,8 @@ public void initialize(RaftServer server, RaftGroupId id,
       super.initialize(server, id, raftStorage);
       this.raftGroupId = id;
       storage.init(raftStorage);
+      LOG.info("OzoneManagerStateMachine is initializing. raftPeerId = {}, raftGroupId = {}",
+          server.getId(), raftGroupId);
```

Comment:
- Print peer id first.  Let's do it for all the messages, it will be easier for debugging test output since there are more than one peers.
- Could you also remove `raftGroupId` and replace the usage with `getGroupId()`?
```java
@@ -88,7 +89,6 @@ public class OzoneManagerStateMachine extends BaseStateMachine {
       new SimpleStateMachineStorage();
   private final OzoneManager ozoneManager;
   private RequestHandler handler;
-  private RaftGroupId raftGroupId;
   private volatile OzoneManagerDoubleBuffer ozoneManagerDoubleBuffer;
   private final ExecutorService executorService;
   private final ExecutorService installSnapshotExecutor;
@@ -134,8 +134,8 @@ public void initialize(RaftServer server, RaftGroupId id,
       RaftStorage raftStorage) throws IOException {
     getLifeCycle().startAndTransition(() -> {
       super.initialize(server, id, raftStorage);
-      this.raftGroupId = id;
       storage.init(raftStorage);
+      LOG.info("{}: initialize {} with {}", id, getGroupId(), getLastAppliedTermIndex());
     });
   } 
```